File: cpp/program/Pembayaran.cpp
#pragma once
#include <string>
using namespace std;

class Pembayaran {
private:
    string id_pembayaran;
    string tanggalPembayaran;
    string metode;
    double jumlah;
public:
    Pembayaran(string id="", string tgl="", string metode="", double jumlah=0.0) {
        this->id_pembayaran = id;
        this->tanggalPembayaran = tgl;
        this->metode = metode;
        this->jumlah = jumlah;
    }

    // Getter & Setter
    string getIdPembayaran() { return id_pembayaran; }
    string getTanggalPembayaran() { return tanggalPembayaran; }
    string getMetode() { return metode; }
    double getJumlah() { return jumlah; }

    void setIdPembayaran(string id) { id_pembayaran = id; }
    void setTanggalPembayaran(string tgl) { tanggalPembayaran = tgl; }
    void setMetode(string m) { metode = m; }
    void setJumlah(double j) { jumlah = j; }
};

File: java/program/Pembayaran.java
public class Pembayaran {
    private String idPembayaran;
    private String tanggalPembayaran;
    private String metode;
    private double jumlah;

    public Pembayaran(String idPembayaran, String tanggalPembayaran, String metode, double jumlah) {
        this.idPembayaran = idPembayaran;
        this.tanggalPembayaran = tanggalPembayaran;
        this.metode = metode;
        this.jumlah = jumlah;
    }

    // Getter
    public String getIdPembayaran() {
        return idPembayaran;
    }

    public String getTanggalPembayaran() {
        return tanggalPembayaran;
    }

    public String getMetode() {
        return metode;
    }

    public double getJumlah() {
        return jumlah;
    }

    // Setter
    public void setIdPembayaran(String idPembayaran) {
        this.idPembayaran = idPembayaran;
    }

    public void setTanggalPembayaran(String tanggalPembayaran) {
        this.tanggalPembayaran = tanggalPembayaran;
    }

    public void setMetode(String metode) {
        this.metode = metode;
    }

    public void setJumlah(double jumlah) {
        this.jumlah = jumlah;
    }
}
